### ISSUE TITLE
[BUG] "No executable path is specified. Pass "executablePath" option directly" after update to Big Sur 11.1

### ISSUE THREAD

**Issue by sjurgis** [microsoft/playwright#4722]
**Context:**
- Playwright Version:   ^1.8.0-next.1608000060809 and latest stable
- Operating System: Big Sur 11.1
- Node.js version: v14.15.1
- Browser: Chromium
- Extra: [any specific details about your environment]

```
  System:
    OS: macOS 11.1
    Memory: 444.80 MB / 32.00 GB
  Binaries:
    Node: 14.15.1 - /usr/local/bin/node
    npm: 6.14.8 - /usr/local/bin/npm
  Languages:
    Bash: 3.2.57 - /bin/bash
```
**Code Snippet**

```javascript
const {
  chromium
} = require('playwright-chromium');
(async () => {
  const browser = await chromium.launch();
  // ...
})();
```

result 
```
(node:11975) UnhandledPromiseRejectionWarning: browserType.launch: No executable path is specified. Pass "executablePath" option directly.
Note: use DEBUG=pw:api environment variable and rerun to capture Playwright logs.
    at Connection.sendMessageToServer (/Users/heyoo/workspace/DND/sfdx/node_modules/playwright-chromium/lib/client/connection.js:69:15)
    at Proxy.<anonymous> (/Users/heyoo/workspace/DND/sfdx/node_modules/playwright-chromium/lib/client/channelOwner.js:44:61)
    at /Users/heyoo/workspace/DND/sfdx/node_modules/playwright-chromium/lib/client/browserType.js:61:73
    at BrowserType._wrapApiCall (/Users/heyoo/workspace/DND/sfdx/node_modules/playwright-chromium/lib/client/channelOwner.js:72:34)
    at BrowserType.launch (/Users/heyoo/workspace/DND/sfdx/node_modules/playwright-chromium/lib/client/browserType.js:52:21)
    at /Users/heyoo/workspace/DND/sfdx/test.js:5:34
    at Object.<anonymous> (/Users/heyoo/workspace/DND/sfdx/test.js:7:3)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
```
**Describe the bug**

Dunno what to add more. Started today I think after update to latest Big Sur. Tried reinstalling node_modules, updating XCode, rebooting, etc to no avail.

**Comment by mxschmitt** [comment]
The beta of macOS Big Sur 11.1 is not supported yet, maybe it can be added. But seems like they want to wait until it's officially released: https://github.com/microsoft/playwright/pull/4699#issuecomment-744315483

**Comment by sjurgis** [comment]
@mxschmitt there might be some confusion with macOS release naming. I definitely didn't update via beta channel - it's actual version is 11.1 (20C69) that is going out to everyone today.

https://developer.apple.com/documentation/macos-release-notes/macos-big-sur-11_1-release-notes

**Comment by AleksandrHorev** [comment]
@mxschmitt macOS Big Sur 11.1 is the latest stable from today https://developer.apple.com/documentation/macos-release-notes or https://apps.apple.com/eg/app/macos-big-sur/id1526878132 
It will be great to support it as soon as possible.
Thank you

**Comment by mxschmitt** [comment]
Thanks for pointing out both. 👍  I created a corresponding patch #4724 here.

**Comment by VincentDondain** [comment]
Hi, I'm experiencing exactly the same issue since I updated to macOS 11.1 yesterday.

`Error: No executable path is specified. Pass "executablePath" option directly.`

I had to manually add `executablePath: "/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome"` to `chromium.launch` but that didn't fix the issue everywhere.

**Comment by sjurgis** [comment]
Quick fix would be adding latest to your package.json `"playwright-chromium": "^1.8.0-next.1608058261773",` until 1.7 gets released.

**Comment by RoXuS** [comment]
supposed to work ?

I got 
![image](https://user-images.githubusercontent.com/3296671/102271120-36bfaf00-3f1f-11eb-96db-793a401a043f.png)


On install with :
````
    "playwright-chromium": "^1.8.0-next.1608058261773",
    "playwright": "^1.8.0-next.1608058261773",
````

**Comment by patrickberkeley** [comment]
I'm seeing this issue again after updating to macos 11.2.1. The suggestions above don't seem to fix the issue.

**Comment by aslushnikov** [comment]
@patrickberkeley This bug has been fixed in Playwright 1.8. What's your playwright version? Can you please file a new issue with all the details so that we can continue discussion there?

**Comment by patrickberkeley** [comment]
@aslushnikov for completeness of this thread, I'm running version 1.8.1, and a new issue is here: https://github.com/microsoft/playwright/issues/5561

**Comment by Yurii397** [comment]
@aslushnikov Hi 
Yesterday updated my iOS to the new one - Monterey Version 12.0.1
and mentioned issue above with `executablePath ` started to appear.  

My current Playwright version - 1.16.1
I've tried to upgrade it to '1.18.0-next-alpha-nov-16-2021' but it doesn't help. I'm using Node js. reinstalling all node js modules doesn't help.

Would be thankful for any updates or suggestions!

**Comment by mxschmitt** [comment]
Please open a new issue, otherwise its hard to follow etc. Thanks!
